Data-Engineering-CYA/README.md

```diff
@@ -0,0 +1,57 @@
+# Choose Your Own Adventure
+Welcome to the adventure into the Data team at Vox Media. Our goal is to become a world-class data engineering team leading the way in developing production-ready and state-of-the-art data science solutions. We do this by providing Vox Media Stakeholders with democratized and integrated data solutions that help them gain clarity, insights, and scale with first-party data. 
+## How To Prepare
+For the Data Engineer role, you will participate in a 45-technical panel interview with the Revenue Data team. Choose any ONE of the following prompts and come prepared to present and discuss. 
```

Comment:
I'm guessing "45-technical" should be "45-minute technical"

Reply:
Maybe also include expectations for presenting and discussing? "come prepared to present and discuss in whichever way you prefer. For example, you can share outlined notes, walkthrough a visual presentation, or engage in detailed conversation." <--- those might not be the right words, but outlining for people is always helpful.